CMake/resolve_dependency_modules/arrow/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ if(VELOX_ENABLE_ARROW)
   set(ARROW_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/arrow_ep")
   set(ARROW_CMAKE_ARGS
       -DARROW_PARQUET=OFF
+      -DARROW_DEPENDENCY_SOURCE=AUTO
```

Comment:
Shouldnt it be AUTO by default ?

Reply:
Kind of, see https://github.com/facebookincubator/velox/issues/10818#issuecomment-2305879175